xml/containers-docker-overview.xml

```diff
@@ -82,7 +82,7 @@
   </listitem>
   <listitem>
    <para>
-    &deng; is not a full virtualization stack like &xen;, &kvm;, or &libvirt;.
+    &deng; is not a full virtualization stack like &xen;, &kvm;, and &libvirt;.
```

Comment:
Here the `and` is definitely wrong, because it implies that only the combination of the three make a full stack.

Reply:
Agreed. I have reverted this instance.